src/Components/test/testassets/ComponentsApp.Server/Pages/Index.cshtml

```diff
@@ -15,5 +15,12 @@
     <app>@(await Html.RenderComponentAsync<App>())</app>
 
     <script src="_framework/components.server.js"></script>
+    <script>
+        Blazor.start({
+            configureSignalR: function (builder) {
+                builder.configureLogging(2); // LogLevel.Information
```

Comment:
I don't have any cool ideas about making this better inside Blazor. @anurse  and I discussed the possibility of accepting strings in this API but that still feels lame.

Other ideas?

Reply:
I think it's fine

People who consume these libraries via TS imports will be able to pass SignalR enum values. People who consume them as a `<script>` tag won't, but that's just the nature of the SignalR API here (SignalR presumably documents what the accepted values are).

Reply:
I don't think we document the constant values though. I think it's reasonable to have the SignalR APIs accept strings though. If that's filed somewhere, we can continue with this for now.

Reply:
logged an issue